presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -834,4 +834,7 @@ Drop a schema::
 Hive Connector Limitations
 --------------------------
 
-:doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
+- :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire
+  partitions.
+- :doc:`/sql/alter-schema` usage fails, since the Hive metastore does not
```

Comment:
`usage fails` -> `is not supported` ?

Reply:
then we use supported twice and it sounds bad as a sentence. I suggest to leave as is.